src/Lykke.Blockchains.Integrations.Contract/TransactionsExecutor/SendingTransactionBuildingError.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Runtime.Serialization;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
+
+namespace Lykke.Blockchains.Integrations.Contract.TransactionsExecutor
+{
+    /// <summary>
+    /// Enum describing reason of the transaction bulding failure.
+    /// </summary>
+    [PublicAPI]
+    [JsonConverter(typeof(StringEnumConverter), typeof(CamelCaseNamingStrategy), new object[0], false)]
+    public enum SendingTransactionBuildingError
```

Comment:
Also, probably, we'll add some generic code for possible integration-specific non-transient integration specific errors

Reply:
At the moment, at the HTTP contact level, this case corresponds to the HTTP status code 400 without errorCode:

> 400 Bad Request: 
> * No additional error code specified - transaction can’t be built with the given parameters and it will be never possible to build the transaction with exactly the same parameters.
> * Additional error code type is sendingTransactionBuildingError.

There is no special error code to exclude ambiguous errors processing.